src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -758,6 +833,10 @@ li {
 
 // Overrides for vuetify components (Checkbox alignment, inputField labels/text size/colour)
 ::v-deep {
+  #btn-remove.v-btn.v-btn--disabled {
+    color: $app-red !important;
+    opacity: .4;
+  }
```

Comment:
In future tickets, we should work towards adding Vuetify overrides like this in the overrides.scss file.

Reply:
Definitely. 
The main issue i experience is the colour override. Can't seem to get it to keep it's inherited colour without being explicit.

Reply:
There are multiple rules for which CSS selector "wins". In this case it's probably a matter of specificity: some other selector is more specific than yours.

This is fine for now. We can improve this kind of thing next time we're here.